CONTRIBUTING.md

```diff
@@ -156,18 +166,6 @@ docker run --rm -e UID=$(id -u) -e GID=$(id -g) \
 ```
 
 
-
-## Update version catalog
```

Comment:
No need to document this explicitly, this is no longer meant to be run by a human 🤖

Reply:
what will we do tomorrow then? 😨

Reply:
Idk... maybe _bombelki_? 👶🏻👶🏻👶🏻